front/app/components/PostShowComponents/SharingModalContent/index.tsx

```diff
@@ -242,43 +168,83 @@ class SharingModalContent extends PureComponent<
       whatsAppMessage
     ) {
       return (
-        <Container className={className}>
-          <Rocket src={rocket} alt="" />
-          <Title className={`e2e-${postType}-social-sharing-modal-title`}>
+        <Box
+          width="100%"
+          display="flex"
+          flexDirection="column"
+          alignItems="center"
+          justifyContent="center"
+          className={className}
+        >
+          <img
```

Comment:
I would use Image component from the component library - also not convinced there's a need for alt text - the img is decorative => it can remain ""